packages/content-type-reply/src/Reply.ts

```diff
@@ -54,26 +47,24 @@ export class ReplyCodec implements ContentCodec<Reply> {
     return {
       type: ContentTypeReply,
       parameters: {
+        // TODO: cut when we're certain no one is looking for "contentType" here.
         contentType: content.contentType.toString(),
         reference: content.reference,
       },
       content: bytes,
     };
   }
 
-  decode(
-    content: EncodedContent<ReplyParameters>,
-    codecs: CodecRegistry,
-  ): Reply {
+  decode(content: EncodedContent, codecs: CodecRegistry): Reply {
     const decodedContent = proto.EncodedContent.decode(content.content);
-    const contentType = ContentTypeId.fromString(
-      content.parameters.contentType,
-    );
-
+    if (!decodedContent.type) {
+      throw new Error("missing content type");
+    }
```

Comment:
how would this error be possible?

Reply:
proto3 encoding permits any object to be absent even if we shouldn't be producing anything without a `type`.